src/nodejs/CMakeLists.txt

```diff
@@ -21,6 +25,14 @@ set_target_properties(node_osrm PROPERTIES CXX_CLANG_TIDY "")
 target_no_warning(node_osrm suggest-destructor-override)
 target_no_warning(node_osrm suggest-override)
 
+# https://github.com/cjntaylor/node-cmake/issues/53#issuecomment-842357457
```

Comment:
We could use https://github.com/cmake-js/cmake-js here which is recommended by Node API docs, but it would require much more refactoring in the way how we build our binaries: they force us to use their own node script to run `cmake` commands what is not very convenient.